pkg/reconciler/kubernetes/tektoninstallerset/controller.go

```diff
@@ -65,6 +68,21 @@ func NewExtendedController() injection.ControllerConstructor {
 			Handler:    controller.HandleAll(impl.EnqueueControllerOf),
 		})
 
+		clusterRoleBindingInformer.Get(ctx).Informer().AddEventHandler(cache.FilteringResourceEventHandler{
```

Comment:
i am not sure why we need to watch on these

Reply:
want to react immediately, if there is change notified on the resources owned by `installerset`.
In this PR, we have introduced separate RBAC and SA for pruner. I want to reconcile them immediately, if there is a change happened.

Reply:
i dont think it is needed, we dont need to react on manual changes immediately, we can wait for next reconcile

Reply:
IMO, the purpose of the operator should keep the desired state of the application all the time as soon as possible.
RBAC/SA needs for `pruner` job. pruner job will fail if something not found.

Reply:
@piyush-garg I have removed this change from this PR. I have handled this via https://github.com/tektoncd/operator/pull/1442